pkg/acs/calacs/calacs/acsmain.c

```diff
@@ -25,7 +26,7 @@ char MsgText[MSG_BUFF_LENGTH]; // Global char auto initialized to '\0'
 
 static void printSyntax(void)
 {
-    printf("syntax:  calacs.e [--help] [-t] [-s] [-v] [-q] [-r] [--version] [--gitinfo] [-1|--nthreads <N>] [--ctegen <1|2>] [--pctetab <path>] input \n");
+    printf("syntax:  calacs.e [--help] [-t] [-s] [-v] [-q] [-r] [--version] [--gitinfo] [-1|--nthreads <N>] [--ctegen <1|2>] [--pctetab <path>] [--forwardModelOnly] input \n");
```

Comment:
What is the use case of running the entire pipeline on RAW using forward model? I don't think this needs to be an option for `calacs.e`?

Reply:
Good point, I'll remove.